test/schema/conditional_workflow.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: beehive/v1
+kind: Workflow
+metadata:
+  name: beehive-deployment
+  labels:
+    app: example2
+spec:
+  strategy:
+    type: sequence
+  template:
+    metadata:
+      name: beehive-deployment
+      labels:
+        app: example
+        use-case: test
+  agents:
+    - agent1
+    - agent2
+    - agent3
+  exception: step3
+  steps:
+    - name: step1
+      agent: agent1
+      condition:
```

Comment:
Is this condition evaluated against the output of the agent1?

Reply:
Can we assume that the condition expression is the python expression for now?

Reply:
Yes, it should be Python and can use `in` and `out` dictionaries

Reply:
> Is this condition evaluated against the output of the agent1?

Against the output of the agent from previous step, yes.

Reply:
When is the `agent1` in this step executed?